### DISCUSSION TITLE
Theme Switcher Help

### DISCUSSION THREAD

**Discussion by scarcekoi** [tailwindlabs/tailwindcss#15507] [Help]
Hello! I'm making a website and using Tailwind CSS. I installed the Catppuccin plugin but I'm wondering if I can have a button on the website that switches the theme between light and dark and have it switch between Mocha and Latte?
[GitHub Repository](https://github.com/scarcekoi/spectralhorizons)

**Comment by JulioGrajales** [comment]
I've never used the Catppuccin plugin before, but based on [their website](https://tailwindcss.catppuccin.com/) it seems it's just a matter of making a button that adds/removes the classnames to the HTML.

For my astro site i used a similar variation of [their recipe](https://docs.astro.build/en/tutorial/6-islands/2/#add-client-side-interactivity).
